tracing-core/src/field.rs

```diff
@@ -224,6 +224,11 @@ pub trait Visit {
         self.record_debug(field, &value)
     }
 
+    /// Visit a byte slice.
+    fn record_byte_slice(&mut self, field: &Field, value: &[u8]) {
```

Comment:
nit, take it or leave it: i might just call this `record_bytes`, personally? but it's fine either way.

Reply:
Sure, I went with your suggestion.